<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drill_chart_xbar_r_tabwidget</class>
 <widget class="QWidget" name="drill_chart_xbar_r_tabwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonSelectXbarR">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select mode</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="gex.qrc">
         <normaloff>:/gex/icons/selectmode.png</normaloff>:/gex/icons/selectmode.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMoveXbarR">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Drag mode: move the view area</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="gex.qrc">
         <normaloff>:/gex/icons/move.png</normaloff>:/gex/icons/move.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonZoomXbarR">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Zoom In: LEFT Click. Zoom Out: RIGHT Click</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="gex.qrc">
         <normaloff>:/gex/icons/drill3d_zoom.png</normaloff>:/gex/icons/drill3d_zoom.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHomeXbarR">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Back to Zoom level 1:1</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="gex.qrc">
         <normaloff>:/gex/icons/home.png</normaloff>:/gex/icons/home.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineSeparatorXBarR">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>26</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTypeXBarR">
       <property name="text">
        <string>Control chart:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTypeControlChart">
       <property name="toolTip">
        <string>Select charting mode: Histogram, Trend, BiVariate, etc...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelViewport">
       <property name="text">
        <string>Samples to plot:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxViewport">
       <property name="toolTip">
        <string>Data samples to chart...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="60,40">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="GexSingleChartWidget" name="XbarChartWidget" native="true"/>
     </item>
     <item>
      <widget class="GexSingleChartWidget" name="RChartWidget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GexSingleChartWidget</class>
   <extends>QWidget</extends>
   <header>gexsinglechartwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
